package.json

```diff
@@ -5,12 +5,13 @@
   "main": "index.js",
   "scripts": {
     "start": "NODE_ENV='development' npm run watch",
-    "build": "npm-run-all build:images build:images-chrome build:js      build:pug      build:sass build:fonts build:Fafonts",
-    "build:beta": "npm-run-all build:images build:images-chrome build:js:beta build:pug:beta build:sass build:fonts build:Fafonts",
+    "build": "npm-run-all build:images build:images-chrome build:js      build:pug      build:sass build:fonts build:Fafonts build:sidenav",
+    "build:beta": "npm-run-all build:images build:images-chrome build:js:beta build:pug:beta build:sass build:fonts build:Fafonts build:sidenav",
     "build:js": "parcel build src/js/chrome.js -d build/js --public-url /apps/chrome/js",
     "build:js:beta": "parcel build src/js/chrome.js -d build/js --public-url /beta/apps/chrome/js",
     "build:images": "cpx 'node_modules/@patternfly/patternfly/assets/images/*' build/assets/images",
     "build:images-chrome": "cpx 'static/images/*' build/assets/images",
+    "build:sidenav": "node src/js/nav/buildNav.js",
```

Comment:
I don't think this is a good idea. It would be much better to download the yaml on fly and generate object from it.

Reply:
I assumed creating it on the build step would be the way to go in order to make it faster. Wouldn't generating it on the fly slow it down each time we navigate to a page?